### ISSUE TITLE
Literature review on scaling laws

### ISSUE THREAD

**Issue by RaymondLi0** [bigcode-project/Megatron-LM#21]
Get an idea of the different flavours of scaling-law works that are out there. Any work that tries to estimate the optimal scale of model and dataset size, with regards to a certain metric (PPL, or other)
Some references:
- Kaplan et al:  [Scaling Laws for Neural Language Models](https://arxiv.org/pdf/2001.08361.pdf)
- Henighan et al: [Scaling Laws for Autoregressive Generative Modeling](https://arxiv.org/pdf/2010.14701.pdf)
- Hernandez et al: [Scaling Laws for Transfer](https://arxiv.org/pdf/2102.01293.pdf)
- Tay et al: [Scale efficiently: Insights from pre-training and
fine-tuning transformers](https://arxiv.org/pdf/2109.10686.pdf)
- Tay et al: [Scaling Laws vs Model Architectures: How does Inductive Bias Influence Scaling?](https://arxiv.org/pdf/2207.10551.pdf)
- Tay et al: [Transcending Scaling Laws with 0.1% Extra Compute](https://arxiv.org/pdf/2210.11399.pdf)
- Hoffmann et al: [Training Compute-Optimal Large Language Models](https://arxiv.org/pdf/2203.15556.pdf)
- Bansal et al: [Data Scaling Laws in NMT: The Effect of Noise and Architecture](https://proceedings.mlr.press/v162/bansal22b/bansal22b.pdf)
- Clark et al: [Unified Scaling Laws for Routed Language Models](https://proceedings.mlr.press/v162/clark22a/clark22a.pdf)

**Comment by RaymondLi0** [comment]
Particularly, it would be good to know if some previous work examined downstream performance instead of PPL in a scaling-law setting

**Comment by keyboardAnt** [comment]
Additional papers:

- Bavarian et al: [Efficient Training of Language Models to Fill in the Middle](https://arxiv.org/pdf/2207.14255.pdf) (OpenAI, Jul 2022)

**Comment by RaymondLi0** [comment]
Other related work:
- Wei et al: https://arxiv.org/pdf/2206.07682.pdf. Although they do not do "scaling laws", they study how capabilities of LLMs  are affected by scale
